doc/Programming-guide/Key-platform-features/Common-Type-Library/index.md

```diff
@@ -45,21 +46,25 @@ Similarly, an attempt to load CT schema with the already used version will resul
 }
 ```
 
-The CTL user interface automatically suggests the next available version (max loaded + 1) for a given FQN when creating a new CT schema.
-CT schema can have dependencies on other CT schemas.
-CT schema dependencies defined as an array of FQN-version combinations (identifiers of referenced CT schemas, see org.kaaproject.sample.ReferencedCT in the example above).
-Deleting CT schema is only permitted if its FQN-version combination is not referenced in any other CT schemas.
+Attempting to load a CT schema with no version or with a used version will result in an error.
+A CT schema can have dependencies on other CT schemas.
+CT schema dependencies are defined as an array of the CT schema IDs (for example, see org.kaaproject.sample.ReferencedCT schema in the code snippet above).
+Deleting a CT schema is only permitted if its ID is not referenced in any other CT schema.
 Cyclic dependencies are not permitted.
-(Thus, CT schemas are nodes in a directed acyclic graph of dependencies.)
+Thus, CT schemas are nodes in a directed acyclic graph of dependencies.
+
+## CT scopes and visibility
 
-# CT scopes and visibility
+CTs can be defined within these scopes: **system**, **tenant**, and **application**.
+Scopes impact the visibility of CTs.
+For example, a CT defined for [application]({{root_url}}Glossary/#kaa-application) A is not visible for application B.
+Any FQN is unique within its scope.
```

Comment:
It's unique in the scope of its CT, but may match FQNs in the other CT scopes as per the table. Fixed.